cmd/nerdctl/inspect.go

```diff
@@ -42,18 +42,35 @@ func newInspectCommand() *cobra.Command {
 	return inspectCommand
 }
 
+var validInspectType = map[string]bool{
+	"container": true,
+	"image":     true,
+	"all":       true,
+}
+
 func addInspectFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("format", "f", "", "Format the output using the given Go template, e.g, '{{json .}}'")
 	cmd.RegisterFlagCompletionFunc("format", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return []string{"json"}, cobra.ShellCompDirectiveNoFileComp
 	})
+	cmd.Flags().String("type", "all", "Specify inspect-object type ('image', 'container' or 'all') (default 'all')")
```

Comment:
The default should be `container` for Docker compatibility

Reply:
Done :+1: